### PR TITLE
StaleElementReferenceException fix

### PR DESCRIPTION
**Pull request by eroshenkoam**

### CODE REVIEW

File: htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/loader/decorator/proxyhandlers/WebElementNamedProxyHandler.java [hunk 1]
```diff
@@ -1,27 +1,61 @@
...
+            } catch (StaleElementReferenceException e) {
+                lasException = e;
+                this.waitFor();
+            }
+        }
+        throw lasException;
+    }
+
+    protected long sleepFor() {
+        return 200L;
```
Comment (ham1) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/loader/decorator/proxyhandlers/WebElementNamedProxyHandler.java]: Is there a way to make this customisable? The default in `Wait` is normally 500ms right?

Comment (eroshenkoam) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/loader/decorator/proxyhandlers/WebElementNamedProxyHandler.java]: Fixed

File: htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/loader/decorator/proxyhandlers/WebElementNamedProxyHandler.java [hunk 2]
```diff
@@ -1,27 +1,61 @@
 package ru.yandex.qatools.htmlelements.loader.decorator.proxyhandlers;
 
+import org.openqa.selenium.StaleElementReferenceException;
 import org.openqa.selenium.support.pagefactory.ElementLocator;
 import org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler;
+import org.openqa.selenium.support.ui.Clock;
+import org.openqa.selenium.support.ui.SlowLoadableComponent;
```
Comment (eroshenkoam) [inline htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/loader/decorator/proxyhandlers/WebElementNamedProxyHandler.java]: Fixed

**Comment by eroshenkoam** [comment]
LGTM?

**Pull request merged**